contentcuration/contentcuration/frontend/channelEdit/vuex/assessmentItem/actions.js

```diff
@@ -1,10 +1,18 @@
-import { sanitizeAssessmentItems } from '../../utils';
```

Comment:
Have we stopped using this function completely? Could it be deleted?

Reply:
Oh yes, I might need it when I work on validation but removing for now.